bin/hsc_gen2_to_gen3.sh

```diff
@@ -0,0 +1,11 @@
+mkdir temp_gen2repo
```

Comment:
Is a shebang intentionally being avoided?  `shellcheck` will complain about it.

Reply:
OK  Will add it.